Gruntfile.js

```diff
@@ -202,17 +186,24 @@ module.exports = function(grunt) {
         tasks: ['copy_dist']
       },
       scripts: {
-        files: ['<%= jshint.files %>', './js/**/*.js'],
+        files: ['<%= jshint.files %>'],
         tasks: ['jshint']
       },
       style: {
-        files: ['<%= jscs.all.src %>', './js/**/*.js'],
+        files: ['<%= jscs.all.src %>'],
```

Comment:
In the past, I’ve liked using `git ls-files` to only target files we are actually tracking but I know this can break down in places where we don’t have Git. In our case, we even have Git on CI though, right? Would you be open to that?

Reply:
Not sure what you're suggesting here. We've already specified, in detail, what the `jscs` task applies to, so we'll be using that for the watch.

Reply:
Apologies if I wasn’t clear. I meant that we often reference `js/**/*.js` but that includes all files in the repo, regardless of whether they’re tracked in Git or not. When working on a new feature, one can have JS files in the project that haven’t been checked in but still trigger lint errors, etc.